typescript/phoenix-sdk/src/market.ts

```diff
@@ -237,15 +208,60 @@ export class Market {
   getExpectedOutAmount({
     side,
     inAmount,
+    slot,
+    unixTimestamp,
   }: {
     side: Side;
     inAmount: number;
+    slot: beet.bignum;
+    unixTimestamp: beet.bignum;
   }): number {
-    return getMarketExpectedOutAmount({
-      marketData: this.data,
-      side,
-      inAmount,
-    });
+    const numBids = toNum(this.data.header.marketSizeParams.bidsSize);
```

Comment:
why can't this call the other implementation of getExpectedOutAmount? prefer to not duplicate this code